src/redux/missions/missionsSlice.ts

```diff
@@ -44,11 +46,31 @@ export const missionsSlice = createSlice({
     initializeMissions: (state, action: PayloadAction<{ missions: Phase[] }>) => {
       state.isInitialized = true;
       state.data.missions = action.payload.missions.map(convertPhaseToUTC);
+
+      // If no mission was loaded or if the previously selected mission was removed,
+      // automatically select first available mission from the updated list
+      const isSelectedMissionLoaded = state.data.missions.some(
+        (mission) => mission.identifier === state.selectedMissionIdentifier
+      );
+      if (!isSelectedMissionLoaded) {
+        state.selectedMissionIdentifier = action.payload.missions[0].identifier;
```

Comment:
What happens if there are no missions... ?  E.g. if all the mission assets were deinitialized